dist/components/app.jsx

```diff
@@ -128,19 +131,23 @@ class App extends React.Component {
           serviceFee: response.data[0].service_fee,
           occupationalFee: response.data[0].occupational_fee,
           nameOfOwner: response.data[0].name_of_owner,
-        });
+        }, () => this.getBooking());
```

Comment:
I am not sure if `this` is going to be preserved inside the `getBooking` when it gets invoked from here.  I might be wrong.  A safer way of writing this line can be:
`}, this.getBooking.bind(this)`.